source/index.rst

```diff
@@ -229,6 +229,15 @@ Community translations can be found in a variety of languages in the bottom-left
 
          C++
 
+      .. button-link:: https://robotpy.readthedocs.io/projects/wpilib/en/stable/api.html
```

Comment:
This should also be updated. https://robotpy.readthedocs.io/projects/robotpy/en/latest/